NuGet/ChustaSoft.Tools.Authorization/Services/SessionService.cs

```diff
@@ -120,6 +143,26 @@ private Session GetUserSession(TUser user)
             return session;
         }
 
+        private void ManageUnsucceededSignin(SignInResult result)
```

Comment:
Create ExtensionMethod for SignInResult doing that logic, coud be used in other places and SessionService specifically shouldn't know how to manage it

Reply:
Created an extension method and moved logic there